app/worker/registration_processor.rb

```diff
@@ -41,6 +44,7 @@ def lane_init(competition_id, user_id)
       empty_registration = Registration.new(attendee_id: "#{competition_id}-#{user_id}",
                                             competition_id: competition_id,
                                             user_id: user_id)
+      RegistrationHistory.create(attendee_id: "#{competition_id}-#{user_id}", entries: [History.new({ 'changed_attributes' => {}, 'actor_user_id' => user_id, 'action' => 'Worker create' })])
```

Comment:
Too much happening on one line. Please split the `History.new` thing into its own line.